node/src/reactor/main_reactor/catch_up.rs

```diff
@@ -269,13 +269,7 @@ impl MainReactor {
             self.chainspec.core_config.simultaneous_peer_requests,
         );
         let leap_status = self.sync_leaper.leap_status();
-        info!(
-            ?block_hash,
-            ?leap_status,
-            "CatchUp: {} {}",
-            block_hash,
-            leap_status
-        );
+        info!("CatchUp: status for {} is {}", block_hash, leap_status);
```

Comment:
I might be alone in this opinion, but I think we should ideally do this the other way round, ie. something like:
`info!(?block_hash, ?leap_status, "CatchUp status");`
This makes better use of structured logging: this way the message itself doesn't ever change, only the variables that are logged on the side do, so it's easier to look for these messages in the logs.

Same applies to other log messages: ideally, we'd have no `{}` or `{:?}` in them, all of the values would be printed using structured logging constructs like `?block_hash`.

(@Fraser999 correct me if I'm wrong!)

Reply:
Yes, makes sense to me. It would be probably easier to parse and correlate with other events if we use structured logging.

Reply:
Changed in https://github.com/casper-network/casper-node/pull/3507/commits/754a7d4d610ed5c667c11451813c7776c62d518b